Patch written by Matteo Bernardini <ponce@slackbuilds.org>
Just followed the gcc hints ;)

diff -Naur kipi-plugins-1.9.0.orig/htmlexport/wizard.cpp kipi-plugins-1.9.0/htmlexport/wizard.cpp
--- kipi-plugins-1.9.0.orig/htmlexport/wizard.cpp	2011-02-28 00:53:44.000000000 +0100
+++ kipi-plugins-1.9.0/htmlexport/wizard.cpp	2012-09-12 07:06:12.279681840 +0200
@@ -78,7 +78,7 @@
 public:
     WizardPage(KAssistantDialog* dialog, const QString& title)
     : QWidget(dialog) {
-        setupUi(this);
+        this->setupUi(this);
         layout()->setMargin(0);
         mPage = dialog->addPage(this, title);
     }
diff -Naur kipi-plugins-1.9.0.orig/printimages/wizard.cpp kipi-plugins-1.9.0/printimages/wizard.cpp
--- kipi-plugins-1.9.0.orig/printimages/wizard.cpp	2011-02-28 00:53:44.000000000 +0100
+++ kipi-plugins-1.9.0/printimages/wizard.cpp	2012-09-12 06:54:49.726668411 +0200
@@ -88,7 +88,7 @@
     WizardPage ( KAssistantDialog* dialog, const QString& title )
         : QWidget ( dialog )
     {
-        setupUi ( this );
+        this->setupUi ( this );
         layout()->setMargin ( 0 );
         mPage = dialog->addPage ( this, title );
     }
